UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -33,7 +33,7 @@ public class LengthTests : LengthTestsBase
 
         protected override double FeetInOneMeter => 3.28084;
 
-        protected override double USSurveyFeetInOneMeter => 3.2808333333465;
+        protected override double USSurveyFeetInOneMeter => 3.2808333333;
```

Comment:
Why would we not want the most accurate factor? What do you mean by "eliminate truncation error"?